### PR TITLE
docs: "Try it first!" section using docker

### PR DESCRIPTION
**Pull request by cristobalgvera**
Add an easy way to try the base configuration of LunarVim in a sandbox environment, allowing users to take a look at the keymappings, colorschemes, which-key setup, etc.

Personally, I find it useful to give this kind of interaction to new users.

### CODE REVIEW

**Comment by Z3rio** [comment]
The code looks good to me at first glance. I will test this in a couple of hours when I get home :)

**Review by Z3rio — APPROVED** [review]

**Comment by vonPB** [comment]
As far as I know you can install lvim next to your own nvim -> without changing your normal config at all.
@kylo252 probably has a better understanding of this.
I think there was some discussion about pointing that out in the docs already.

Otherwise, never worked with docker, but looks alright :)

**Comment by Nojipiz** [comment]
I think is a cool feature, it's true that you can install lvim next to your own nvim, but it will require to _"foul the system"_ with another installation, docker provides predictable and isolated environment. Please give it a try!

**Review by vonPB — CHANGES_REQUESTED** [review]
Got a few problems with hyper-v on my machine, so cant test it ;/
I'm happy to merge this if someone can confirm it works.

Also feel like the code block is a bit lengthy/blocking. Would be nice if we could wrap it in something like the [detail container from vuepress](https://vuepress.vuejs.org/guide/markdown.html#custom-containers). Couldn't find one in Docusaurus. Happy to write on another day...

**Comment by kylo252** [comment]
could you guys update the dockerfile in the main repo and link to it here instead?

it could even be a new installation method.

File: docs/01-installation.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Installation
 
+## Try it first!
+
+If your are using Docker in your machine, you can run the following command to try LunarVim while keeping your existing Neovim configuration intact.
```
[the remark this answers is not in the document]
Comment (cristobalgvera) [reply docs/01-installation.md]: I rephrase the detail, let me know how it looks!

**Comment by cristobalgvera** [comment]
> Also feel like the code block is a bit lengthy/blocking. Would be nice if we could wrap it in something like the [detail container from vuepress](https://vuepress.vuejs.org/guide/markdown.html#custom-containers). Couldn't find one in Docusaurus. Happy to write on another day...

Yeah, seems like Docusaurus doesn't provide it :( do you have any other idea about how to present it?

**Comment by vonPB** [comment]
as kylo said, we can update (or add a new one) the docker file in [lunarvim](https://github.com/LunarVim/LunarVim/blob/master/utils/docker/Dockerfile.remote). After that we could just have a one liner to run the "Try it first". That would be small enough for my taste :)

@cristobalgvera can you try if the docker file works as u expect and let me know? I'll update the file to get either `rolling` or `release-1.2/neovim-0.8` instead of the old `rolling` in a sec.

**Comment by vonPB** [comment]
new text looks good! Ty

**Comment by vonPB** [comment]
https://github.com/LunarVim/LunarVim/blob/update_docker/utils/docker/Dockerfile.remote
leaving the change here for now. I'll merge it after your feedback if it works :)

**Comment by LostNeophyte** [comment]
> https://github.com/LunarVim/LunarVim/blob/update_docker/utils/docker/Dockerfile.remote leaving the change here for now. I'll merge it after your feedback if it works :)

It does work, tested in podman

**Comment by kylo252** [comment]
we should add it as a new tab btw

image

**Comment by cristobalgvera** [comment]
Guys, I don't know how to continue, what should I do?

Thanks!

**Comment by vonPB** [comment]
@cristobalgvera  since https://github.com/LunarVim/LunarVim/blob/master/utils/docker/Dockerfile.remote is basically doing what you initially wrote, it would make sense to just add a command to the docs that gets this file and executes it (similar to the installation command). This wouldn't be taking up as much space as your initial proposal did and we could just add it as a tab here:
image
with something like "Try it first in docker!"

**Comment by vonPB** [comment]
Thanks for not abandoning this :)!

**Comment by cristobalgvera** [comment]
> @cristobalgvera since https://github.com/LunarVim/LunarVim/blob/master/utils/docker/Dockerfile.remote is basically doing what you initially wrote, it would make sense to just add a command to the docs that gets this file and executes it (similar to the installation command). This wouldn't be taking up as much space as your initial proposal did and we could just add it as a tab here: image with something like "Try it first in docker!"

Here is the result 😁

image

**Comment by vonPB** [comment]
@cristobalgvera  looks great!
Can you maybe remove the line breaks?  A one liner would look in my opinion.
image

Happy to merge otherwise :)

**Review by vonPB — APPROVED** [review]

**Pull request merged**